src/components/createPlaylist/CreatePlaylist.jsx

```diff
@@ -1,148 +1,147 @@
-import React, { Component } from 'react'
-import withStyles from '@material-ui/core/es/styles/withStyles'
-import { loadCSS } from 'fg-loadcss'
-import Grid from '@material-ui/core/Grid'
-import Typography from '@material-ui/core/Typography'
-import Card from '@material-ui/core/Card'
-import CardActionArea from '@material-ui/core/CardActionArea'
-import CardContent from '@material-ui/core/CardContent'
-import ReactPath from '../../lib/ReactPath'
-import classnames from 'classnames'
-import ScreenShareTwoToneIcon from '@material-ui/icons/ScreenShareTwoTone'
-import CreateTwoToneIcon from '@material-ui/icons/CreateTwoTone'
-import VideocamTwoToneIcon from '@material-ui/icons/VideocamTwoTone'
+// import React, { Component } from 'react'
+// import withStyles from '@material-ui/core/es/styles/withStyles'
+// import { loadCSS } from 'fg-loadcss'
+// import Grid from '@material-ui/core/Grid'
+// import Typography from '@material-ui/core/Typography'
+// import Card from '@material-ui/core/Card'
+// import CardActionArea from '@material-ui/core/CardActionArea'
+// import CardContent from '@material-ui/core/CardContent'
+// import ReactPath from '../../lib/ReactPath'
+// import classnames from 'classnames'
+// import ScreenShareTwoToneIcon from '@material-ui/icons/ScreenShareTwoTone'
+// import CreateTwoToneIcon from '@material-ui/icons/CreateTwoTone'
+// import VideocamTwoToneIcon from '@material-ui/icons/VideocamTwoTone'
 
-const styles = theme => ({
-   root: {
-      flexGrow: 1,
-   },
-   card: {
-      maxWidth: 275,
-      // fontSize: '6rem'
-   },
-   bullet: {
-      display: 'inline-block',
-      margin: '0 2px',
-      transform: 'scale(0.8)'
-   },
-   icon: {
-      width: theme.spacing(10),
-      height: theme.spacing(10),
-      marginTop: theme.spacing(4),
-      marginBottom: theme.spacing(4)
-   },
-   gradient: {
-      background: 'linear-gradient(45deg, #f09433 0%,#e6683c 25%,#dc2743 50%,#cc2366 75%,#bc1888 100%)',
-      fallbacks: [
-         {
-            background: '-webkit-linear-gradient(45deg, #f09433 0%,#e6683c 25%,#dc2743 50%,#cc2366 75%,#bc1888 100%)'
-         },
-         {
-            background: '-moz-linear-gradient(45deg, #f09433 0%, #e6683c 25%, #dc2743 50%, #cc2366 75%, #bc1888 100%)'
-         },
-         {
-            background: '#f09433'
-         }
-      ],
-      filter: 'progid:DXImageTransform.Microsoft.gradient( startColorstr=\'#f09433\', endColorstr=\'#bc1888\',GradientType=1 )'
-   }
-})
+// const styles = theme => ({
+//    root: {
+//       flexGrow: 1,
```

Comment:
Remove

Reply:
yes, you should remove the unrelated code before sending PR